splunk_add_on_ucc_framework/commands/rest_builder/endpoint/base.py

```diff
@@ -79,9 +88,13 @@ def generate_conf_with_default_values(self) -> str:
 
     def generate_rh(self) -> str:
         fields = []
+        special_fields = []
         for field in self._fields:
             field_line = field.generate_rh()
             fields.append(field_line)
+        for special_field in self._special_fields:
+            field_line = special_field.generate_rh()
```

Comment:
nit: I think here we can use list comprehension, or we can remove redundant variable declaration.